UnityProject/Assets/Scripts/Items/Weapons/Gun.cs

```diff
@@ -188,11 +191,13 @@ public class Gun : NetworkBehaviour, IPredictedCheckedInteractable<AimApply>, IC
 		public ItemSlot pinSlot;
 		public ItemSlot suppressorSlot;
 
+		protected float PinRemove = 10f;
```

Comment:
is this one a constant?

Reply:
yea

Reply:
give it the ``const`` modifier then. Would we ever want to have guns with different pinRemove time? Also yeah, ``pinRemoveTime`` is a better name for the variable unless I misunderstood its use.